backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/controller/external/VerkuendungController.java

```diff
@@ -15,11 +20,14 @@
 public class VerkuendungController {
 
   private final StoreNormendokumentationspaketUseCase storeNormendokumentationspaketUseCase;
+  private final LoadNormendokumentationspacketProcessingStatusUseCase statusService;
 
   public VerkuendungController(
-    StoreNormendokumentationspaketUseCase storeNormendokumentationspaketUseCase
+    StoreNormendokumentationspaketUseCase storeNormendokumentationspaketUseCase,
+    LoadNormendokumentationspacketProcessingStatusUseCase statusService
```

Comment:
Nitpicky but I would align with the naming we always have for the use cases, this one is called `statusService` though it is not a service but a use case. Was confusing to read in the endpoint below